### PR TITLE
cmd, cask: info prints whether installed

### PR DESCRIPTION
**Pull request by willfaught**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew typecheck` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Fixes https://github.com/Homebrew/brew/issues/17050

### CODE REVIEW

File: Library/Homebrew/cmd/tap-info.rb [hunk 1]
```diff
@@ -64,6 +64,7 @@ def print_tap_info(taps)
             puts unless i.zero?
             info = "#{tap}: "
             if tap.installed?
+              info += "Installed"
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/tap-info.rb:67]: What's the output if this is not installed?

**Review by MikeMcQuaid — COMMENTED** [review]
Thanks for the PR @willfaught, looking great so far! Interested in thoughts from other @Homebrew/maintainers here.

**Comment by SMillerDev** [comment]
I don't look at brew enough to understand where this would print. Do you have an example output @willfaught ?

File: Library/Homebrew/cmd/tap-info.rb (hunk 1 above)
Comment (razvanazamfirei) [inline Library/Homebrew/cmd/tap-info.rb:67]: This needs a whitespace after `Installed`, but the terminology here is a bit misleading as we're showing the number of items in the tap, not the number of items that are installed.

Comment (SMillerDev) [reply Library/Homebrew/cmd/tap-info.rb:67]: "Available" could be better

Comment (MikeMcQuaid) [reply Library/Homebrew/cmd/tap-info.rb:67]:
```console
$ brew tap-info some/tap
some/tap: Not installed
```

**Review by MikeMcQuaid — APPROVED** [review]

File: Library/Homebrew/cmd/tap-info.rb [hunk 2]
```diff
@@ -64,6 +64,7 @@ def print_tap_info(taps)
...
               info += if (contents = tap.contents).blank?
                 "no commands/casks/formulae"
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/tap-info.rb:69]:
```suggestion
                "\nNo commands/casks/formulae"
```

File: Library/Homebrew/cmd/tap-info.rb [hunk 3]
```diff
@@ -64,6 +64,7 @@ def print_tap_info(taps)
...
               else
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/tap-info.rb:70]: @willfaught below this line change:
- `contents.join(", ")` to `"\n#{contents.join(", ")}"`
- `info += ", private" if tap.private?` to `info += "\nPrivate" if tap.private?`

**Review by MikeMcQuaid — COMMENTED** [review]
@willfaught Nice work so far, a few tweaks and this should be good to go!

File: Library/Homebrew/cmd/tap-info.rb (hunk 3 above)
Comment (willfaught) [reply Library/Homebrew/cmd/tap-info.rb:70]: [29c561e](https://github.com/Homebrew/brew/pull/17076/commits/29c561eb595a1410394342d79cf897ff1657fb44)

File: Library/Homebrew/cmd/tap-info.rb (hunk 1 above)
Comment (willfaught) [reply Library/Homebrew/cmd/tap-info.rb:67]: @razvanazamfirei What kind of whitespace do you mean? The original string value appended didn't have trailing whitespace.

@SMillerDev The original code checks `tap.installed?`, and the else block uses "Not installed", so wouldn't "Installed" be best to match?

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @willfaught!

**Comment by willfaught** [comment]
Cool! Thanks for helping to test, @MikeMcQuaid! Happy to help.

**Pull request merged**